tests/unit/test_collections.py

```
@@ -0,0 +1,210 @@
+            assert moved.collection == target.id
+
+            client.collections.pages.remove(workspace_slug, target.id, page_collection_id)
+            listed_after = client.collections.pages.list(workspace_slug, target.id)
+            assert not any(
+                row.page and row.page.get("id") == page.id for row in listed_after.results
+            )
+        finally:
+            try:
+                client.pages.delete_workspace_page(workspace_slug, page.id)
+            except Exception:
+                pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)